_docs/performance-and-scalability/overview.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Overview
+overview: Provides a conceptual introduction to Istio's Performance and Scalability
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+
+We have a 4 pronged approach for Istio's performance characterization, tracking and improvements:
```

Comment:
s/We have/The Performance and Scalability Working Groups has/

Reply:
changed though it's a bit verbose and I'm not sure users care about which part if istio is doing what